src/Models/Config.php

```diff
@@ -101,16 +105,17 @@ public static function getValue(
 
         $websiteId = $scope === ConfigScopes::SCOPE_STORE ? Rapidez::getStore($scopeId)['website_id'] : $scopeId;
 
-        $result = static::query()
+        $resultObject = static::query()
             ->withoutGlobalScope('scope-fallback')
             ->where('path', $path)
             ->where(fn ($query) => $query
                 ->when($scope === ConfigScopes::SCOPE_STORE, fn ($query) => $query->whereStore($scopeId))
                 ->when($scope !== ConfigScopes::SCOPE_DEFAULT, fn ($query) => $query->orWhere(fn ($query) => $query->whereWebsite($websiteId)))
                 ->orWhere(fn ($query) => $query->whereDefault())
             )
-            ->first('value')
-            ?->value;
+            ->first('value');
+
+        $result = $resultObject ? $resultObject->value : ($options['default'] ?? null);
```

Comment:
I think this could cause trouble with the cache function, in case the default value should differ.
Maybe force `$options['cache']` to false if the result did not come from the DB

Reply:
I don't know if I necessarily like abusing a boolean here to differentiate between the cases, as it feels a bit arbitrary and risky when it comes to possible future refactorings, but I do get the issue.

I'm trying to think what would be a better option though, in a lot of cases I can imagine we actually just want to eventually refactor the defaults to already exist within Rapidez so you don't have to specify them yourself 🤔

Reply:
A separate local variable is fine too. We just cannot cache the passed default value as it is not in-line with the actual data in the database. 
Caching it would mean we remove the choice of a differing default value.

Reply:
For now I've just accepted that caching `false` is the best solution, because we do still want to actually cache the config path even if the actual value was not found.